.github/workflows/test.yml

```diff
@@ -60,11 +78,11 @@ jobs:
       - uses: actions-rs/toolchain@v1
         with:
           toolchain: stable
-      - uses: actions-rs/cargo@v1
-        with:
-          command: test
-          args: --no-default-features --manifest-path ts-rs/Cargo.toml
-      - uses: actions-rs/cargo@v1
-        with:
-          command: test
-          args: --all-features --manifest-path ts-rs/Cargo.toml
\ No newline at end of file
+      - name: No features
+        run: |
+          cargo test --no-default-features
+          tsc ts-rs/tests-out/**/*.ts --noEmit --noUnusedLocals
+      - name: All features
+        run: |
+          cargo test --all-features
+          tsc ts-rs/tests-out/**/*.ts --noEmit --noUnusedLocals
```

Comment:
Turns out this doesn't work as-is. `/**/` Seems to only match one subdirectory, but it isn't recursive by default.  
I assume this is running on bash, where it can be enabled with `shopt -s globstar`. Commit is on the way.